server/mailet/droplists-cassandra/extensions.properties

```diff
@@ -0,0 +1 @@
+guice.extension.module=org.apache.james.droplists.cassandra.CassandraDropListsModule
```

Comment:
Maybe we could move this file into a `sample-configuration` sub folder to make the intent clear?